dockerspawner/dockerspawner.py

```diff
@@ -54,8 +54,72 @@ def validate(self, obj, value):
 class DockerSpawner(Spawner):
     """A Spawner for JupyterHub that runs each user's server in a separate docker container"""
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._init_deprecated_methods()
+
+    def _init_deprecated_methods(self):
```

Comment:
Unlike the Authenticators, where subclassing and overriding methods is a standard approach, I don't think subclassing and overriding these methods is something we need to keep on DockerSpawner.

Reply:
It makes sense. Thanks for explaining this, Min!